package.json

```diff
@@ -16,18 +16,18 @@
     "link-packages": "./scripts/link-packages.sh",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
-    "lint:changelogs": "yarn workspaces run lint:changelog",
+    "lint:changelogs": "cross-env npm_package_repository_url=https://github.com/MetaMask/snaps-skunkworks.git yarn workspaces foreach -pt run lint:changelog",
```

Comment:
Will we be able to get rid of `cross-env` once `auto-changelog` supports Yarn 3?

Reply:
Yeah, it's a temporary workaround.

Reply:
What do you think about using the full option names instead of `-pt`?